### ISSUE TITLE
中文语音同步效果比较差，能否考虑升级语音模型到deepspeech最新版本

### ISSUE THREAD

**Issue by zhangziliang04** [MRzzm/DINet#2]
感谢作者，非常棒的工作，实践中发现，中文语音同步效果比较差，能否考虑升级语音模型到deepspeech最新版本或者其他可以适配中文的版本。谢谢。我的微信：13718542435，希望能和作者交流。

**Comment by davidmartinrius** [comment]
Did you try it? Did it work after updating deepshepch?

**Comment by zhangziliang04** [comment]
中文兼容性，是目前最大的问题所在，目前两个思路：1. 上deepspeech最新版本，尤其是支持中文的版本；2. 换用其他的支持中文的版本。  面临的问题：神经网络结构需要重新设计，所有的与训练模型，需要重新来过。是个浩大的工程，而且不一定能有效果。正在尝试中。。。

**Comment by MRzzm** [comment]
您好，感谢您的关注，我现在没有时间继续这个工作，您可以按照自己的想法做实验试一下哈。

**Comment by zhangziliang04** [comment]
感谢回复，打算基于这个项目进行重构，主要思路是：1. 替换语音模块（支持中文），2. 修改网络模型结构：（支持中文）。想要咨询一下，完成一轮训练的实验环境需求和大致的时间消耗，提前心里有个底。谢谢。@MRzzm

**Comment by zhangziliang04** [comment]
> Did you try it? Did it work after updating deepshepch?

进行中，发现缺了部分内容。你可以加我，一起看看。

**Comment by RoseWeHere** [comment]
> > Did you try it? Did it work after updating deepshepch?
> 
> 进行中，发现缺了部分内容。你可以加我，一起看看。

您好， 我也再进行类似工作，能留下联系方式交流一下嘛

**Comment by magicleo** [comment]
@zhangziliang04 请问有新的进展了吗？中文效果怎么样了？

**Comment by zhangziliang04** [comment]
缺判别器的，代码开源的不全。暂时搁置了。

**Comment by 91xiaoyang** [comment]
Great project, can this be done in real-time? If possible, how should I modify it?

**Comment by 12Song12** [comment]
请问这个代码是不全吗？ @91xiaoyang

**Comment by alphonz** [comment]
请问这个代码是不全吗？ @zhangziliang04

**Comment by cucdengjunli** [comment]
mark

**Comment by jiayueshiliuqi** [comment]
> 

判别器我这有了，你要再尝试一下吗？

**Comment by XJX777** [comment]
> > 
> 
> 判别器我这有了，你要再尝试一下吗？

**Comment by XJX777** [comment]
> > 
> 
> 判别器我这有了，你要再尝试一下吗？

您好我目前正在复现这部分工作，请问判别器这部分您有code或者可参考的code吗？感谢

**Comment by foxyear-kyumin** [comment]
> > 
> 
> 判别器我这有了，你要再尝试一下吗？

可以分享一下吗？

**Comment by jiayueshiliuqi** [comment]
> > > 
> > 
> > 
> > 判别器我这有了，你要再尝试一下吗？
> 
> 可以分享一下吗？

这个项目的训练我搁置了,需要先解决声音编码的部分,syncnet的训练你可以参考wav2lip的

**Comment by jiayueshiliuqi** [comment]
> > > 
> > 
> > 
> > 判别器我这有了，你要再尝试一下吗？
> 
> 您好我目前正在复现这部分工作，请问判别器这部分您有code或者可参考的code吗？感谢

这个项目的训练我搁置了,需要先解决声音编码的部分,syncnet的训练你可以参考wav2lip的

**Comment by create-li** [comment]
[PaddlePaddle-DeepSpeech](https://github.com/yeyupiaoling/PaddlePaddle-DeepSpeech)
这个训练好的中文语音模型，请问尝试过吗？

**Comment by wning13** [comment]
试了下改用meta的hubert模型效果还可以

**Comment by jinwonkim93** [comment]
> I tried switching to meta’s hubert model and the effect was ok.

did you train syncnet from scratch?

**Comment by wning13** [comment]
> > I tried switching to meta’s hubert model and the effect was ok.
> 
> did you train syncnet from scratch?

Yes, I modified the syncnet architecture, so I need to train it from scratch

**Comment by tailangjun** [comment]
> 试了下改用meta的hubert模型效果还可以

大佬，改用 hubert后，网络结构需要怎样修改，麻烦给点建议哈，谢谢~

**Comment by jinwonkim93** [comment]
> > > I tried switching to meta’s hubert model and the effect was ok.
> > 
> > 
> > did you train syncnet from scratch?
> 
> Yes, I modified the syncnet architecture, so I need to train it from scratch

@wning13 Cool did you change the training loss for syncnet? i guess infoNCE loss is also good choice for better representation.
it would be wonderful if you can share me the training code you used. if possible.

**Comment by wning13** [comment]
> > > > I tried switching to meta’s hubert model and the effect was ok.
> > > 
> > > 
> > > did you train syncnet from scratch?
> > 
> > 
> > Yes, I modified the syncnet architecture, so I need to train it from scratch
> 
> @wning13 Cool did you change the training loss for syncnet? i guess infoNCE loss is also good choice for better representation. it would be wonderful if you can share me the training code you used. if possible.

Sorry, I can't share my code at the moment. I used BCE loss (similar to what WAV2Lip uses); perhaps using InfoNCE could yield better results. I'll try it out later (currently, my GPU is tied up running other training tasks). If you've experimented with this, could you share your results?
